src/pages/guide/javascript/quickstart.md

```diff
@@ -12,6 +12,12 @@ And then run it as usual:
 cd my-first-app
 npm run build
 ```
+You can also build in watch mode: 
+```sh
+npm run build -w
```

Comment:
looks like `npm run watch` will do this - could you change?
see https://github.com/BuckleScript/bucklescript/blob/master/jscomp/bsb/bsb_templates.ml#L191